packages/rn-tester/js/examples/TextInput/TextInputExample.ios.js

```diff
@@ -945,6 +948,42 @@ if (Platform.OS === 'macos') {
         return <AutoCorrectSpellCheckGrammarCheckCallbacks />;
       },
     },
+    {
+      title: 'Clear text on submit - Multiline Textfield',
+      render: function (): React.Node {
+        return (
+          <View>
+            <Text>Default submit key (Enter):</Text>
+            <TextInput
+              multiline={true}
+              clearTextOnSubmit={true}
+              style={styles.multiline}
+            />
+            <Text>Custom submit key (Enter): - same as above</Text>
+            <TextInput
+              multiline={true}
+              clearTextOnSubmit={true}
+              style={styles.multiline}
+              submitKeyEvents={[{key: 'Enter'}]}
+            />
+            <Text>Custom submit key (CMD + Enter):</Text>
+            <TextInput
+              multiline={true}
+              clearTextOnSubmit={true}
+              style={styles.multiline}
+              submitKeyEvents={[{key: 'Enter', metaKey: true}]}
+            />
+            <Text>Custom submit key (Shift + Enter):</Text>
+            <TextInput
+              multiline={true}
+              clearTextOnSubmit={true}
+              style={styles.multiline}
+              submitKeyEvents={[{key: 'Enter', shiftKey: true}]}
```

Comment:
If/when an equivalent of `IHandledKeyboardEvent` is ported to RN-macOS, I guess it can share a type. I have no plans at the moment to do that, so not an ask.

Reply:
Ok.
So I leave it as is atm